### PR TITLE
Se 449 create GitHub workflow for necessist run

### PR DESCRIPTION
**Pull request by bfcoursewool**
Everything for automating necessist. Includes a dockerfile for the necessist build, github workflow for building and pushing to ECR as well as redeploying the k8s job from the new container, and a k8s manifest file describing the new necessist job. 

Run the workflow from here in github under Actions -> Run Necessist, and when it finally finishes (the GH workflow itself only takes 5 minutes) there will be a new necessist-YY-mm-dd.db file in the "necessist database" bucket of S3 in our AWS account. Voila!

### CODE REVIEW

**Review by bcamp136 — APPROVED** [review]

**Review by oscarsernarosero — COMMENTED** [review]
Looks good to me Rob. I didn't really mess at all with Necessist, so Shane will be the real test.

File: foundry.toml [hunk 1]
```diff
@@ -5,12 +5,15 @@ libs = ['lib']
 
 [profile.local]
 eth-rpc-url = 'http://localhost:8545'
+code-size-limit = 0x60000
```
Comment (ShaneDuncan602) [inline foundry.toml]: I don't think we need the 0x for this value. We should be fine with actually using the kb value of 45000 or so.

File: Dockerfile.anvil [hunk 2]
```diff
@@ -13,4 +13,4 @@ ENV CHAIN_ID 31337
 
 RUN pip install -r requirements.txt
 
-ENTRYPOINT anvil --host 0.0.0.0 --chain-id $CHAIN_ID
\ No newline at end of file
+ENTRYPOINT anvil --host 0.0.0.0 --code-size-limit 0x60000 --chain-id $CHAIN_ID
```
Comment (ShaneDuncan602) [inline Dockerfile.anvil]: We could probably leave out the 0x value and just use the actual Kb

File: necessist-run.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+source ~/.bashrc
+source ~/.cargo/env
+source .venv/bin/activate
+cargo install necessist
+necessist --verbose --framework foundry -- --ffi
```
Comment (ShaneDuncan602) [inline necessist-run.sh:8]: ToB had to give us a fix for a problem that we encountered. We have to use

`cargo install --git https://github.com/trailofbits/necessist --branch fix-foundry-build`

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
Just a couple of questions

**Review by ShaneDuncan602 — APPROVED** [review]
Great job

**Pull request merged**